src/client/eppo-client.ts

```diff
@@ -131,40 +132,32 @@ export default class EppoClient {
   private assignmentCache?: AssignmentCache;
   // whether to suppress any errors and return default values instead
   private isGracefulFailureMode = true;
-  private expectObfuscated: boolean;
-  private obfuscationMismatchWarningIssued = false;
-  private configObfuscatedCache?: boolean;
   private requestPoller?: IPoller;
   private readonly evaluator = new Evaluator();
   private configurationRequestor?: ConfigurationRequestor;
   private readonly overrideValidator = new OverrideValidator();
 
   constructor({
     eventDispatcher = new NoOpEventDispatcher(),
-    isObfuscated = false,
+    isObfuscated,
     flagConfigurationStore,
     banditVariationConfigurationStore,
     banditModelConfigurationStore,
     overrideStore,
     configurationRequestParameters,
-  }: {
-    // Dispatcher for arbitrary, application-level events (not to be confused with Eppo specific assignment
-    // or bandit events). These events are application-specific and captures by EppoClient#track API.
-    eventDispatcher?: EventDispatcher;
-    flagConfigurationStore: IConfigurationStore<Flag | ObfuscatedFlag>;
-    banditVariationConfigurationStore?: IConfigurationStore<BanditVariation[]>;
-    banditModelConfigurationStore?: IConfigurationStore<BanditParameters>;
-    overrideStore?: ISyncStore<Variation>;
-    configurationRequestParameters?: FlagConfigurationRequestParameters;
-    isObfuscated?: boolean;
-  }) {
+  }: EppoClientParameters) {
     this.eventDispatcher = eventDispatcher;
     this.flagConfigurationStore = flagConfigurationStore;
     this.banditVariationConfigurationStore = banditVariationConfigurationStore;
     this.banditModelConfigurationStore = banditModelConfigurationStore;
     this.overrideStore = overrideStore;
     this.configurationRequestParameters = configurationRequestParameters;
-    this.expectObfuscated = isObfuscated;
+
+    if (isObfuscated !== undefined) {
+      logger.info(
+        `[Eppo SDK] specifying isObfuscated no longer has an effect; obfuscation is inferred from the configuration.`,
```

Comment:
minor: do you think the users will get the message and what they need to do? I think "obfuscation is inferred from the configuration" might be confusing?

Maybe something along the lines of this?
```suggestion
        `[Eppo SDK] isObfuscated init option is deprecated and will be removed in the next major release; obfuscation is now inferred automatically, so you can safely remove the option.`,
```

Reply:
applied, thanks!